portals/publisher/source/src/app/components/Apis/Details/LifeCycle/LifeCycleUpdate.jsx

```diff
@@ -315,28 +370,37 @@ class LifeCycleUpdate extends Component {
                     )}
                     <ScopeValidation resourcePath={resourcePath.API_CHANGE_LC} resourceMethod={resourceMethod.POST}>
                         <div className={classes.buttonsWrapper}>
-                            {!isWorkflowPending
-                                && lifecycleButtons.map((transitionState) => {
-                                    /* Skip when transitions available for current state ,
-                            this occurs in states where have allowed re-publishing in prototype and published sates */
-                                    return (
-                                        <Button
-                                            disabled={transitionState.disabled
+                            {(api.advertiseInfo && api.advertiseInfo.advertised) ? (
```

Comment:
Bring back the support for the API life cycle

Reply:
Added back with https://github.com/wso2/apim-apps/pull/130/commits/a4ec374ee6e7fbc43128130106cb656cd00db3f5